### ISSUE TITLE
[a11y]: Coachmark drag functionality is not accessible via keyboard

### ISSUE THREAD

**Issue by anamikaanu96** [carbon-design-system/ibm-products#9030]
The Coachmark drag functionality is not working for keyboard users
